src/main/java/ezschedule/logic/commands/RecurCommand.java

```diff
@@ -23,13 +23,15 @@ public class RecurCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Recurs event in the scheduler by "
             + "the index number used in the displayed event list. "
             + "\nParameters: INDEX (must exist)\n "
-            + CliSyntax.PREFIX_DATE + "ENDING DATE\n "
-            + CliSyntax.PREFIX_EVERY + "{DAY, WEEK, MONTH, YEAR} "
+            + CliSyntax.PREFIX_DATE + "ending date\n "
+            + CliSyntax.PREFIX_EVERY + "{day, week, month} "
             + "\nExample: \n" + COMMAND_WORD + " 1 "
             + CliSyntax.PREFIX_DATE + "2024-02-20 "
-            + CliSyntax.PREFIX_EVERY + "MONTH ";
+            + CliSyntax.PREFIX_EVERY + "month ";
 
```

Comment:
Can you go and remove the all the "CliSyntax" for all the above.
Import the respective CliSyntax.PREFIX_DATE and etc, thus you only need to use PREFIX_DATE and etc directly